### ISSUE TITLE
No break between heading and description in settings row

### ISSUE THREAD

**Issue by steverep** [home-assistant/frontend#11682]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

In all instances of the `<ha-settings-row>` element I surveyed, `span` elements are slotted into both the heading and description.  The component does not seem to put any break or space between them.  The result visually may be fine, but programmatically, and thus what a screen reader sees, is the two pieces of text as one with no break.

In some instances, the right solution seems to be to slot in block elements instead (e.g. `<h#>` and `<p>`), but in others a simple ": " makes much more sense and would be much more screen reader friendly, so I think the fix should be evaluated individually.

### Describe the behavior you expected

I expect some space, maybe a ":", or at least a break 💔 

### Steps to reproduce the issue

1.
2.
3.
...

### What version of Home Assistant Core has the issue?

2022.02.6

**Comment by steverep** [comment]
This is still an issue.

**Comment by steverep** [comment]
Still an issue

**Comment by steverep** [comment]
Still and issue
